diagrams/orego.jl

```diff
@@ -16,39 +15,78 @@ test_sol = TestSolution(sol)
 abstols = 1 ./ 10 .^ (5:8)
 reltols = 1 ./ 10 .^ (1:4);
 setups = [Dict(:alg=>Rosenbrock23()),
-          Dict(:alg=>Rodas3()),
-          Dict(:alg=>TRBDF2())]
+          Dict(:alg=>TRBDF2()),
+          Dict(:alg=>CVODE_BDF()),
+          Dict(:alg=>lsoda())]
 wp = WorkPrecisionSet(prob,abstols,reltols,setups;
                       save_everystep=false,appxsol=test_sol,maxiters=Int(1e5))
-DIAGRAMS["OregoHighTol1"] = plot(wp)
+DIAGRAMS["Orego-HighTol-Group1-1"] = plot(wp)
 
 
 wp = WorkPrecisionSet(prob,abstols,reltols,setups;dense = false,verbose=false,
                       appxsol=test_sol,maxiters=Int(1e5),error_estimate=:l2)
-DIAGRAMS["OregoHighTol2"] = plot(wp)
+DIAGRAMS["Orego-HighTol-Group1-2"] = plot(wp)
 
 wp = WorkPrecisionSet(prob,abstols,reltols,setups;
                       appxsol=test_sol,maxiters=Int(1e5),error_estimate=:L2)
-DIAGRAMS["OregoHighTol3"] = plot(wp)
+DIAGRAMS["Orego-HighTol-Group1-3"] = plot(wp)
 
 # Low Tolerance
 
 abstols = 1 ./ 10 .^ (7:13)
 reltols = 1 ./ 10 .^ (4:10)
 
-setups = [Dict(:alg=>GRK4A()),
-          Dict(:alg=>Rodas4P()),
-          Dict(:alg=>Rodas4())
-]
 wp = WorkPrecisionSet(prob,abstols,reltols,setups;
                       save_everystep=false,appxsol=test_sol,maxiters=Int(1e5))
-DIAGRAMS["OregoLowTol1"] = plot(wp)
+DIAGRAMS["Orego-LowTol-Group1-1"] = plot(wp)
 
 wp = WorkPrecisionSet(prob,abstols,reltols,setups;verbose=false,
                       dense=false,appxsol=test_sol,maxiters=Int(1e5),error_estimate=:l2)
-DIAGRAMS["OregoLowTol2"] = plot(wp)
+DIAGRAMS["Orego-LowTol-Group1-2"] = plot(wp)
 
 wp = WorkPrecisionSet(prob,abstols,reltols,setups;
                       appxsol=test_sol,maxiters=Int(1e5),error_estimate=:L2)
-DIAGRAMS["OregoLowTol3"] = plot(wp)
+DIAGRAMS["Orego-LowTol-Group1-3"] = plot(wp)
```

Comment:
we're going to need more than just OREGO to determine regressions

Reply:
Yeah they are going to be almost same, I just edited this to get a review